src/handlebars/support.handlebars

```diff
@@ -59,6 +59,360 @@
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span></td>
+				</tr>
+				<tr>
+					<td>SUSE Linux Enterprise Server (SLES) 12</td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+					<td></td>
+				</tr>
+				<tr>
+					<td>Ubuntu 16.04</td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span></td>
+				</tr>
+				<tr>
+					<td>Ubuntu 18.04</td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span> <i class="fa fa-star-o" style="color:purple" aria-hidden="true"></i><span class="sr-only">OpenJ9 VM</span></td>
+					<td><i class="fa fa-star-o" style="color:orange" aria-hidden="true"></i><span class="sr-only">Hotspot VM</span></td>
+				</tr>
+			</tbody>
+		</table>
+
+
+		<p><strong>Note:</strong> Not all of these distributions are tested, but Linux distributions that have a minimum glibc version 2.12 on x86 or version glibc version 2.17 on other architectures are expected to function without problems.</p>
```

Comment:
I think we edit this statement.  If someone discovers that we don't work on a listed platform, that's a bug.  Perhaps it would be OK to say that "Any other Linux distro that has a minimum ......... are expected to function without problems."

Reply:
Same in OpenJDK 11

Reply:
Done